style.css

```diff
@@ -88,21 +75,377 @@ h3 {
   line-height: 24px;
   color: #fefcfc;
 }
-/* stylelint-disable-next-line no-descending-specificity */
+
 .social-media a {
   padding: 1rem;
   color: white;
   font-size: 1.5rem;
   transition: 3s;
 }
 
+.nav a {
```

Comment:
As the previous reviewer had suggested, i recommend that you align this social media icons to match figma design. For aesthetic purposes and also develop your skill to clone projects

- [ ] Your design

<img width="370" alt="Screenshot 2022-09-26 at 13 24 00" src="https://user-images.githubusercontent.com/48546992/192254851-fccaf9cc-d77e-46c8-ac62-4c7c5d61f929.png">

- [ ] figma design
<img width="185" alt="Screenshot 2022-09-26 at 13 29 06" src="https://user-images.githubusercontent.com/48546992/192255034-155785f5-7bc1-4aa2-acd7-9739c5c2129a.png">